django_netjsongraph/base/node.py

```diff
@@ -40,50 +45,30 @@ def clean(self):
             self.properties = {}
 
     def save(self, *args, **kwargs):
-        self._format_addresses()
         super(AbstractNode, self).save(*args, **kwargs)
 
-    def _format_addresses(self):
-        """
-        Ensure address format is correct: ";addr1;addr2;addr3;"
-        """
-        self.addresses = self.addresses.replace(',', ';')\
-                                       .replace(' ', '')\
-                                       .replace(';', ';')
-        if not self.addresses.startswith(';'):
-            self.addresses = ';' + self.addresses
-        if not self.addresses.endswith(';'):
-            self.addresses += ';'
-
-    def truncate_addresses(self):
-        """
-        ensures "addresses" field is not too long
-        """
-        max_length = self._meta.get_field('addresses').max_length
-        if len(self.addresses) <= max_length:
-            return
-        addresses = self.address_list
-        # +1 stands for the character added in self._format_address()
-        while len(';'.join(addresses)) + 2 > max_length:
-            addresses.pop()
-        self.addresses = ';'.join(addresses)
+    def view_addresses(self):
+        return "\n".join(self.address_list)
 
     @cached_property
     def address_list(self):
-        addresses = self.addresses.replace(' ', '')
-        if addresses.startswith(';'):
-            addresses = addresses[1:]
-        return addresses[0:-1].split(';')
+        addresses_array = []
+        for address in list(self.addresses.all()):
+            addresses_array.append(address.address)
+        return addresses_array
 
     @property
     def netjson_id(self):
         if self.addresses:
```

Comment:
this will be always `True` with the new change. A possible alternative may be `if self.addresses.count()`